docs/superbench-config.mdx

```diff
@@ -0,0 +1,389 @@
+---
+id: superbench-config
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# SuperBench Config File
+
+[YAML](https://yaml.org/spec/1.2/spec.html) format configuration file is an efficient method to take full advantage of SuperBench.
+You can put it in any place and specify the path to config file through `-c /path/to/config.yaml` in `sb` CLI.
+
+This document covers schema of SuperBench configuration YAML file.
+You can learn YAML basics from [Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml/).
+SuperBench configuration supports most of the YAML features, including anchors and aliases, merge key, etc.
+
+## Conventions
+
+Here lists syntax conventions used in this document:
+
+* The schema and example are in YAML format.
+
+* In YAML mappings which use a colon `:` to mark `key: value` pair.
+
+  The left side of colon is a literal keyword defined in configuration,
+  if it is surrounded by `${}`, like `${name}`, then the key is a string that can be defined by user.
+
+  The right side of colon is a data type, which may be Python built-in types (like `string`, `dict`),
```

Comment:
=> `The right side of colon is a data value, whose type may be Python built-in types`?

Reply:
it's data type, not data value

Reply:
You mean the right side must be data type term, like 'string', 'int', 'dict'? can't be exact data value, like 'enable', 'yes', '32'?

Reply:
yes, it's the convention for this document

Reply:
[an example](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#conventions)